config/style/topo-raster.json

```diff
@@ -0,0 +1,55 @@
+{
+  "id": "st_topo-raster",
+  "layers": [
+    {
+      "filter": ["all", ["==", "class", "dock"]],
+      "id": "Background",
+      "layout": { "visibility": "visible" },
+      "minzoom": 0,
+      "paint": { "background-color": "rgb(208, 231, 244)" },
+      "type": "background"
+    },
+    {
+      "id": "TOPO-RASTER",
```

Comment:
this is quite different to "hillshade" could we be consistent between this style and the other ones in this folder particullary the topographic and topolite with both of the layer names and the source names.